src/components/Ball.js

```diff
@@ -0,0 +1,18 @@
+import { useEffect } from "react";
+
+const Balls = ({numbers, setNumbers, addZeroToUnityNumber, fillArrayWithCharacter}) => {
+
```

Comment:
Po co w propsach przesyłasz addZeroToUnityNumber, fillArrayWithCharacter a deklarujesz w komponencie Container, chociaż tam tego nie używasz ?

Jeżeli to funkcje pomocnicze to wynieś je do osobnego pliku i zaimportuj gdzie potrzebujesz